content/livesync/how-livesync-works.textile

```diff
@@ -0,0 +1,69 @@
+---
+title: How LiveSync works
+meta_description: "LiveSync enables you to synchronize changes in your database to frontend clients at scale."
+product: livesync
+---
+
+LiveSync is a powerful realtime data synchronization feature designed to facilitate broadcasting realtime updates from backend databases to frontend clients at scale. Leveraging Pub/Sub Channels, LiveSync ensures that data updates are propagated reliably and in order to all connected clients in realtime.
+
+LiveSync can be used for realtime collaboration, including collaborative form editing, online auctions, task management applications, or simple multiplayer turn-based games, where maintaining data consistency and low latency are important. Furthermore, developers can also build highly responsive and interactive applications with the optimistic updates design pattern offered out of the box.
+
+LiveSync has three key components, 1) a Models SDK that facilitates optimistic updates for the publisher and helps merge confirmed changes from the backend with the overall frontend state 2) a Database Connector at the backend that can watch changes in your database table using the outbox pattern and broadcast those updates and 3) the default Ably Pub/Sub channels that form the mechanism behind the scenes to broadcast those updates.
+
+<a href="@content/diagrams/what-is-livesync.png" target="_blank">
+  <img src="@content/diagrams/what-is-livesync.png" style="width: 100%" alt="What is LiveSync">
+</a>
+
+h2(#channel-based-broadcasting). Channel-based broadcasting
+
+The mechanism for synchronizing data updates across devices using Ably's LiveSync uses Ably's "pub/sub channels":https://ably.com/docs/channels to publish realtime messages to subscribers. When a message is published on an Ably channel, it is immediately broadcasted to all subscribers of that channel. Similarly, with LiveSync, data updates added to the outbox table are then published to all subscribed clients in realtime, ensuring seamless data synchronization from the database to all clients.
```

Comment:
The first sentence here is a little muddled with 'using' and 'uses'. 

Broadcasted --> broadcast